packages/sit-onyx/src/components/OnyxBadge/OnyxBadge.vue

```diff
@@ -24,16 +25,19 @@ defineSlots<{
       'onyx-truncation-ellipsis',
       'onyx-text',
       `onyx-badge--${props.variation}`,
+      props.dot ? 'onyx-badge--dot' : '',
```

Comment:
```suggestion
      { 'onyx-badge--dot': props.dot },
```

Reply:
This does not work here, because we use an array of classes instead of an object :)
You can read the difference here between array and object binding: https://vuejs.org/guide/essentials/class-and-style#binding-html-classes